runbotics-orchestrator/src/main/java/com/runbotics/domain/NotificationBot.java

```diff
@@ -0,0 +1,108 @@
+package com.runbotics.domain;
+
+import javax.persistence.*;
+import java.io.Serializable;
+import java.time.ZonedDateTime;
+
+@Entity
+@Table(name = "notification_bot")
+public class NotificationBot implements Serializable {
+
+    private static final long serialVersionUID = 1L;
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "sequenceGenerator")
+    @SequenceGenerator(name = "sequenceGenerator")
+    private Long id;
+
+    @Column(name = "user_id", insertable = false, updatable = false)
+    private Long userId;
+
+    @Column(name = "bot_id", insertable = false, updatable = false)
+    private Long botId;
+
+    @ManyToOne
+    private User user;
+
+    @ManyToOne
+    private Bot bot;
```

Comment:
cant you merge fields `user` with `userId` (same for bot)?
we dont need to store ids separately